test/Grpc.IntegrationTests/IntegrationTestBase.cs

```diff
@@ -43,9 +43,9 @@ void IDisposable.Dispose()
         GC.SuppressFinalize(this);
     }
 
-    protected async Task<HostTestLifetime> StartWorkerAsync(Action<IDurableTaskWorkerBuilder> configure)
+    protected async Task<HostTestLifetime> StartWorkerAsync(Action<IDurableTaskWorkerBuilder> configure, Action<IDurableTaskClientBuilder>? clientConfigure = null)
```

Comment:
Let's also rename `configure` to `workerConfigure` to make it less ambiguous.

Reply:
Done.